.aws/src/event-rules/prospect-events/prospectEventRules.ts

```diff
@@ -41,42 +50,75 @@ export class ProspectEvents extends Resource {
     super(scope, name);
 
     // pre-existing queues (prod and dev) created by prospect-api
-    this.sqs = new sqs.DataAwsSqsQueue(
-      this,
-      `prospect-${config.environment}-queue`,
-      {
-        name: `ProspectAPI-${config.environment}-Sqs-Translation-Queue`,
-      }
+    this.sqs = this.createSqsForProspectEvents(
+      this.sqsIdForProspectGenerationEvent,
+      this.sqsNameForProspectGenerationEvent
     );
 
-    this.sqsDlq = new sqs.DataAwsSqsQueue(
-      this,
-      `prospect-${config.environment}-dlq`,
-      {
-        name: `ProspectAPI-${config.environment}-Sqs-Translation-Queue-Deadletter`,
-      }
+    // create a dlq for all Prospect events
+    this.sqsDlq = this.createSqsForProspectEvents(
+      this.dlqIdForProspectEvents,
+      this.dlqNameForProspectEvents
```

Comment:
doesn't need to block as i don't want to bike shed this forever, but maybe moving these special case calls into the `createprospectGenerationEventRule` function?

sub nit-pick: `retrieveSqsForProspectEvents` may be a better name, as this service is _not_ creating those sqs resources (they are created/owned by prospect api - which is confusing and non-standard 😭 ).